Cesium3DTilesContent/test/TestPntsToGltfConverter.cpp

```diff
@@ -238,6 +249,8 @@ TEST_CASE("Converts point cloud with RGBA to glTF") {
   Model& gltf = *result.model;
 
   CHECK(gltf.hasExtension<CesiumGltf::ExtensionCesiumRTC>());
+  CHECK(gltf.isExtensionUsed(ExtensionCesiumRTC::ExtensionName));
+  CHECK(gltf.isExtensionRequired(ExtensionCesiumRTC::ExtensionName));
   CHECK(gltf.nodes.size() == 1);
```

Comment:
Doesn't hurt to add an extra check for the scene, too. (I can do that)

```suggestion
  CHECK(gltf.scenes.size() == 1);
  CHECK(gltf.nodes.size() == 1);
```